docs/tools/vdb_table/data/singlestoredb.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "SingleStoreDB",
+  "links": {
+    "docs": "https://docs.singlestore.com/",
+    "github": "https://github.com/singlestore-labs",
+    "website": "https://www.singlestore.com/",
+    "vendor_discussion": "",
+    "poc_github": "",
+    "slug": "singlestoredb"
+  },
+  "oss": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "",
+    "comment": ""
+  },
+  "dev_languages": {
+    "value": [
+      "c++",
+      "go"
+    ],
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_launch_year": 2017,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-vector-data/",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-vector-data/",
+    "comment": ""
+  },
+  "facets": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Supported through SQL aggregations"
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/reference/sql-reference/data-types/geospatial-types/",
```

Comment:
Any examples how this works for the vector search queries? Can you score results by a combination of vector similarity and geo distance?

Reply:
Yes, that's possible by ordering by the desired geospatial metric (https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-geospatial-features/) and vector distance (https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-vector-data/) in a query. You could combine the metrics or 

At a high level it would look something like this:

```
SELECT 
    id,
    name,
    GEOGRAPHY_DISTANCE(location, @search_point) AS geo_distance,
    vector_embedding <*> @query_vector AS vector_similarity
FROM 
    your_table
WHERE 
    GEOGRAPHY_DISTANCE(location, @search_point) <= @max_distance
ORDER BY 
    geo_distance ASC, 
    vector_similarity DESC
```

Reply:
Thanks for elaborating, but the way I read your query example is "order by geo distance and for equidistant results, order by vector similarity". I had something like "finance jobs near me" in mind, where you need a blended score between the two. Once that is possible, we consider the geo spatial queries fully integrated with vector search.

This is the part I had in mind:
```
ORDER BY 
    geo_distance ASC, 
    vector_similarity DESC
```

Reply:
Thanks for the callout. A blended score where you determine how to weight geo/vector distance beforehand is also possible:

```
SELECT 
    id,
    name,
    GEOGRAPHY_DISTANCE(location, @search_point) AS geo_distance,
    vector_embedding <*> @query_vector AS vector_similarity,
    (1 - (GEOGRAPHY_DISTANCE(location, @search_point) / @max_distance)) * @geo_weight +
    (vector_embedding <*> @query_vector) * @vector_weight AS blended_score
FROM 
    your_table
WHERE 
    GEOGRAPHY_DISTANCE(location, @search_point) <= @max_distance
ORDER BY 
    blended_score DESC
```

Reply:
Got it, that's neat! However, it would I assume require a full-pass through the database, instead of using an index - correct?

That said, we can leave it as full!